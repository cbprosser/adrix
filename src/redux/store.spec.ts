import { store } from '.';

describe('store Suite', () => {
  it('Should have the correct properties', () => {
    expect(store.getState()).toMatchSnapshot();
  });
});
