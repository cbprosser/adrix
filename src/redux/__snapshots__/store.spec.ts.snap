// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`store Suite Should have the correct properties 1`] = `
Object {
  "cards": Object {
    "cards": Array [],
    "requests": Object {
      "searchForCard": Object {
        "status": "idle",
      },
    },
  },
}
`;
